src/transformers/models/idefics/image_processing_idefics.py

```diff
@@ -146,6 +147,7 @@ def preprocess(
         #     transforms.ToTensor(),
         #     transforms.Normalize(mean=image_mean, std=image_std),
         # ])
+        # TODO: Alazar figure out tf version for below
```

Comment:
Unfinished TODO here! Also, you can remove the `# TODO: this converts to torch tensors - switch to convert_to_tensors once it becomes available` below now that we actually support other tensor types.

Reply:
Got it!